### PR TITLE
feat: added report bug button

### LINKED ISSUE #1178: Add Report a Bug panel
- [ ] Add report a bug button in More Actions
- [x] Create Github Issues form for Bugs in the issues repository
- [ ] Hook slack notification to `#unity-support` channel.

Form should contain the next information
- Kernel Version (Injected)
- Renderer Version (Injected)
- System specs: (CPU, Memory, GPU, SO) (Injected)

- Description
- Steps to reproduce

### PR DESCRIPTION
**Pull request by Kinerius**
## What does this PR change?

Fix #1178 

## How to test the changes?

Use this url
https://play.decentraland.zone/?renderer-branch=feat/report-bug&kernel-branch=feat/renderer-config-versions

- Open taskbar menu
- Click on Report Bug
- Form should be filled with unity version, kernel version, nametag, system specs

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by sandrade-dcl — CHANGES_REQUESTED** [review]
Testing the build, when I click on the "Report Bug" button, nothing happens!

image

**Comment by sandrade-dcl** [comment]
> Testing the build, when I click on the "Report Bug" button, nothing happens!
> 
> image

Ok, I didn't say anything... The problem was that I was playing the build only with the unity renderer version, without taking into account the other PR you have in the kernel side. Adding the `&kernel-branch=feat/renderer-config-versions` in the URL, the "Report Bug" button is working 👍

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarMoreMenu.cs [hunk 1]
```diff
@@ -206,4 +217,35 @@ private void ToggleHideUI()
...
+    }
+
+    private string GetVersion(string raw)
+    {
+        var match = Regex.Match(raw, VERSION_PATTERN);
+        if (!match.Success)
+            return raw;
+
+        return match.Value;
+    }
```
Comment (sandrade-dcl) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarMoreMenu.cs:250]: Applying the resposibility principle, I think these 2 methods shouldn't be in the `TaskbarMoreMenu` class. Both methods are very related to "report a bug" and we should have a specific controller for that.

**Review by sandrade-dcl — CHANGES_REQUESTED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarMoreMenu.cs (hunk 1 above)
Comment (Kinerius) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarMoreMenu.cs]: You are right, that class was already convoluted

**Review by sandrade-dcl — APPROVED** [review]

**Pull request merged**